### ISSUE TITLE
Better build instructions or upload an installer

### ISSUE THREAD

**Issue by draeftl0re** [HaxeFoundation/hashlink#400]
Navigating your windows install guide is like navigating a minefield in Vietnam... it doesn't work period. I have tried to compile it from source which appears to be the only option and it fails. Doesn't matter what program I use it never has worked.  I have tried multiple times in the past.  The only way I have ever gotten the heaps engine to work is the base way for a web based game, which I guess i would just need to find a way to convert html games to an executable, but that seems a little bad for a language that says it's optimized for performance across platforms.

**Comment by Gama11** [comment]
>I have tried to compile it from source which appears to be the only option

No, there's binaries here: https://github.com/HaxeFoundation/hashlink/releases

**Comment by ConfidantCommunications** [comment]
Same issue here on an Apple Silicon M1. The make command fails. I finally had to download the "release" binaries. Fortunately running "make install" was verbose and helped me to discern where to put the binaries so they would install. After that, I had to try a couple rounds of running hl from the command line then going into the System Preferences>Security & Privacy panel to approve the unidentified binary and dyll that need to run. Finally could compile and run via terminal after that.
